lib/galaxy/web/base/interactive_environments.py

```diff
@@ -162,16 +162,19 @@ def docker_cmd(self, env_override={}):
         conf = self.get_conf_dict()
         conf.update(env_override)
         env_str = ' '.join(['-e "%s=%s"' % (key.upper(), item) for key, item in conf.items()])
-        return '%s run %s -d %s -p %s:%s -v "%s:/import/" %s' % \
+        volume_str = ' '.join(['-v "%s"' % volume for volume in volumes])
```

Comment:
Rather than just using strings - what do you think about using this ``DockerVolume`` data structure used by the tool/job stuff: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/tools/deps/docker_util.py#L17.

And then maybe add a method to ie_request like 

```
def volume(from, to, **kwds):
   return galay.tools.deps.docker_util.DockerVolume(from, to, **kwds)
```

So ies wouldn't need to import or realize they depend on stuff in that other module?

This feels more structured to me - what do you think?